cpp/src/gandiva/node.h

```diff
@@ -223,7 +223,22 @@ template <typename Type>
 class InExpressionNode : public Node {
  public:
   InExpressionNode(NodePtr eval_expr, const std::unordered_set<Type>& values)
-      : Node(arrow::boolean()), eval_expr_(eval_expr), values_(values) {}
+      : Node(arrow::boolean()), eval_expr_(eval_expr), values_(values),
```

Comment:
Instead of adding redundant precision and scale properties, since this is a templated class its better to separate out the Decimal class and only add precision/scale to them

Reply:
done